NEXT documentation (#1273)
neo4j/docs-cypher: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

label:functionality[]
label:new[]

[source, cypher, role="noheader"]
----
RETURN 1 AS a

NEXT

RETURN 1 AS b
----

> | Introduction of `NEXT` which allows for linear composition of queries into a sequence.
> For more information, see xref:queries/composed-queries/sequential-queries.adoc[].

Reviewer: ```suggestion
| New `NEXT` keyword used for linear composition of queries.
For more information, see xref:queries/composed-queries/sequential-queries.adoc[].
```

Reviewer: The examples below seem to link the pahe in the keyword name.

Author: what do you mean?
